diff --color -urN vacm-2.0.5a.orig/clients/flim/configure.in vacm-2.0.5a/clients/flim/configure.in
--- vacm-2.0.5a.orig/clients/flim/configure.in	2001-05-15 04:55:51.000000000 +0200
+++ vacm-2.0.5a/clients/flim/configure.in	2020-10-05 00:06:02.947648168 +0200
@@ -25,7 +25,7 @@
 
 dnl Add the languages which your application supports here.
 ALL_LINGUAS="ja ko"
-AM_GNU_GETTEXT
+AM_GNU_GETTEXT([external])
 
 SSL_LIBS=""
 AC_ARG_ENABLE(ssl, [  --enable-ssl            build with OpenSSL support],
@@ -71,5 +71,5 @@
 Makefile
 src/Makefile
 plugins/Makefile
-intl/Makefile po/Makefile.in])
+po/Makefile.in])
 
